libs/postgres_ffi/src/lib.rs

```diff
@@ -10,16 +10,39 @@
 use utils::lsn::Lsn;
 
 macro_rules! postgres_ffi {
-    ($version:ident) => {
+    ("v14") => {
```

Comment:
If we're duplicating all the code for v14 and v15, might as well remove the macro and use the contents of the macro directly. But ISTM the macro approach would still work, why the duplication?